src/global/less/corporate-ui/button-groups.less

```diff
@@ -2,7 +2,7 @@
 
   &-vertical {
 
-    > .btn:first-child:not(:last-child) {
+    > .btn {
```

Comment:
this won't override 

`.btn-group-vertical > .btn:last-child:not(:first-child)`

Reply:
> .btn {
      &:first-child:not(:last-child),
      &:last-child:not(:first-child) {
        border-radius: 0;
      }
    }